data/MichaelLarocca.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "Michael Larocca",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Front-End Web Developer | Writer | Musician",
+  "avatar": "https://github.com/MichaelLarocca.png",
+  "tags": [
+    "frontend",
+    "javascript",
+    "writing",
+    "music"
+  ],
+  "socials": [
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/michael-larocca-9315a3202/" },
+    { "icon": "FaTwitter", "url": "https://twitter.com/MikeJudeLarocca" },
+    { "icon": "FaGithub", "url": "https://github.com/MichaelLarocca" }
+  ],
+  "testimonials": [
+    "HOLD",
+    "HOLD",
+    "HOLD"
```

Comment:
This property for `testimonials` can be removed if not being used

Reply:
Ok. I created a template to make the first pull request easier. I will remove it and add it back later when appropriate. 👍🏻